src/data-access/repositories/test/mongo-part-repository.spec.ts

```diff
@@ -86,16 +87,12 @@ describe(`${MongoPartRepository.name}`, () => {
     })
 
     it('does not deletes any pieces, when nonexistent segmentId is given', async () => {
-      const mongoConverter: MongoEntityConverter = mock(MongoEntityConverter)
       const nonExistingId: string = 'nonExistingId'
       const mongoPart: MongoPart = createMongoPart({})
       await testDatabase.populateDatabaseWithParts([mongoPart])
       const db = testDatabase.getDatabase()
 
-      when(mongoConverter.convertParts(anything())).thenReturn([])
-      const testee = createTestee({
-        mongoConverter: mongoConverter,
-      })
+      const testee = createTestee({ })
```

Comment:
If you set the parameter of `createTestee` to have the default value `{}`, then you don't have to pass in an empty object, when using the default state.

Reply:
Done that now.